python/lsst/analysis/tools/tasks/associatedSourcesTractAnalysis.py

```diff
@@ -20,39 +20,36 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 from __future__ import annotations
 
+import numpy as np
+import pandas as pd
+from lsst.geom import Box2D
 from lsst.pipe.base import connectionTypes as ct
 
 from .base import AnalysisBaseConfig, AnalysisBaseConnections, AnalysisPipelineTask
 
-# These need to be updated for this analysis context
-# from ..analysisPlots.analysisPlots import ShapeSizeFractionalDiffScatter
-# from ..analysisPlots.analysisPlots import Ap12PsfSkyPlot
-# from ..analysisMetrics.analysisMetrics import ShapeSizeFractionalMetric
-
 
 class AssociatedSourcesTractAnalysisConnections(
     AnalysisBaseConnections,
     dimensions=("skymap", "tract"),
     defaultTemplates={
-        "inputName": "isolated_star_sources",
-        # "associatedSourcesInputName": "isolated_star_sources"},
+        "outputName": "isolated_star_sources",
```

Comment:
Interesting that you included a default for `outputName`. I don't think we've been doing this elsewhere in `analysis_tools`, but it seems like good practice.